src/core/utils/helpers/general.ts

```diff
@@ -93,6 +94,21 @@ export const getCoverTint = (index: number) => {
   return undefined;
 };
 
+export const getColors = () => {
+  return getConf("colors", configuration);
+};
+
+export const getThresholds = () => {
+  return getConf("thresholds", configuration);
+};
+
+export const daysBetweenTodayAndDate = (date: string) => {
+  const inputDate = dayjs(date);
+  const today = dayjs();
+
+  return Math.ceil(inputDate.diff(today, "day", true));
```

Comment:
Why do we tell `dayjs` to return a floating point number which is then rounded when [the library will itself will return an integer by default](https://day.js.org/docs/en/display/difference)?

Reply:
I'll change it